lib/core/src/display/shape/primitive/glsl/shape.glsl

```diff
@@ -19,6 +19,16 @@ BoundingBox bounding_box (float w, float h) {
     return BoundingBox(-w,w,-h,h);
 }
 
+BoundingBox bounding_box (vec2 size) {
+    float w2 = size.x / 2.0;
+    float h2 = size.y / 2.0;
+    return BoundingBox(-w2,w2,-h2,h2);
+}
+
+BoundingBox inverse (BoundingBox a) {
+    return BoundingBox(0.0,0.0,0.0,0.0);
```

Comment:
So the special value of BoundingBox need a comment. And I propose to make "constructor", like:
```glsl
BoundingBox infinite_bbox() {
    return BoundingBox(0.0,0.0,0.0,0.0);
}
```
So here it will be:
````glsl
BoundingBox inverse (BoundingBox a) {
    return infinite_bbox();
}
```
 and everything is clear now.

Reply:
As discussed via phone, inversed bounding box is infinite and thus does not have reperesentaiton in GLSL. Im addign docs there.